### DISCUSSION TITLE
How to delete dangling cluster entries

### DISCUSSION THREAD

**Discussion by Sheco** [gravitational/teleport#13294] [Q&A] [answered]
Hello, I was testing creating trusted clusters and now I have a few dangling clusters in the root node.

In the WebUI, I can see these leftovers in the Manage Clusters section and in the CLUSTER: select field at the top of the page.

I can't seem to find a way to remove those entries.

**Comment by Sheco** [comment, marked as answer]
Nevermind, I found it.

```
tctl get rc
tctl rm rc/$name
```

Thanks!

**Reply by isaacloven** [reply]
Thanks for sharing your solution!
